scripts/prepdocs.py

```diff
@@ -314,6 +314,29 @@ def refresh_openai_token():
         openai.api_key = token_cred.get_token("https://cognitiveservices.azure.com/.default").token
         open_ai_token_cache[CACHE_KEY_CREATED_TIME] = time.time()
 
+# recursively read the directory to allow for a subdirectory structure
+def read_files(location, use_vectors):
```

Comment:
Could you type this to make it clear whether location is a string or a path-like object?

Reply:
Should be fine now now, renamed from `location` to `pathPattern`